src/main/scala/io/iohk/ethereum/vm/Int256.scala

```diff
@@ -0,0 +1,201 @@
+package io.iohk.ethereum.vm
```

Comment:
Putting it in a different package doesn't mean we'll be using it elsewhere. With the package structure as is we'll have a cyclic dependency between `domain` and `vm`. Just saying.